timelock-server/build.gradle

```diff
@@ -39,11 +45,44 @@ dependencies {
     testCompile group: 'org.mockito', name: 'mockito-core'
     testCompile project(":atlasdb-tests-shared")
 
-    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing'
+    integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing', version: libVersions.timelock_dropwizard
 }
 
 distribution {
     serviceName "timelock-server"
     mainClass 'com.palantir.atlasdb.timelock.TimeLockServerLauncher'
     args 'server', 'var/conf/timelock.yml'
+    defaultJvmOpts "-Xms512m",
```

Comment:
@jboreiko I don't have strong preference if its in this PR or subsequent, but we should make sure it makes the same release.